apps/storybook-app/stories/templates/SolutionsTemplate.stories.tsx

```diff
@@ -0,0 +1,110 @@
+            name: "logo-pagopa",
+            width: 200,
+            height: 200,
+            ext: ".svg",
+            mime: "image/svg",
+          },
+          tags: [{ label: "IO" }, { label: "PagoPA" }, { label: "SEND" }],
+        },
+        {
+          name: "Multe per violazioni al Codice della Strada",
+          description:
+            "La soluzione dedicata all’ottimizzazione dei processi legati all’intero ciclo di vita del servizio multe, dall'emissione del preavviso fino all'invio dell'eventuale contravvenzione.",
+          slug: "multe-per-violazioni-al-codice-della-strada",
+          logo: {
+            url: "https://cdn.developer.pagopa.it/app_Io_107fb95dac.svg",
+            name: "logo-pagopa",
+            width: 200,
+            height: 200,
+            ext: ".svg",
+            mime: "image/svg",
+          },
+          tags: [{ label: "IO" }, { label: "PagoPA" }, { label: "SEND" }],
+        },
+      ],
+    },
+    feature: {
+      items: [
+        {
+          iconName: "MarkEmailRead",
+          subtitle:
+            "I cittadini hanno un accesso più diretto e immediato alle informazioni importanti e ai servizi che possono utilizzare",
+          title: "Comunicazione tempestiva e trasparente",
+        },
+        {
+          iconName: "MarkEmailRead",
+          subtitle:
+            "I cittadini hanno un accesso più diretto e immediato alle informazioni importanti e ai servizi che possono utilizzare",
+          title: "Comunicazione tempestiva e trasparente",
+        },
+        {
+          iconName: "MarkEmailRead",
+          subtitle:
+            "I cittadini hanno un accesso più diretto e immediato alle informazioni importanti e ai servizi che possono utilizzare",
+          title: "Comunicazione tempestiva e trasparente",
+        },
+      ],
+      title: "I benefici della trasformazione digitale dei servizi",
+    },
+    successStories: {
+      title: "Storie di successo",
+      subtitle:
+        "Le testimonianze di come gli enti hanno dato forma a servizi efficienti e accessibili, ottimizzando i processi grazie all’integrazione con le piattaforme PagoPA.",
+      stories: [
+        {
+          title: "Il processo di riscossione della TARI del Comune di Cagliari",
+          image: {
+            url: "https://developer.pagopa.it/images/io-sign-firmare-documento.png",
+            alternativeText: "alternativeText",
+          },
+          publishedAt: new Date(),
+          path: "/solutions/tari-comune-di-cagliari",
+        },
+        // {
+        //   title: "Il processo di riscossione della TARI del Comune di Cagliari",
+        //   image: {
+        //     url: "https://developer.pagopa.it/images/io-sign-firmare-documento.png",
+        //     alternativeText: "alternativeText",
+        //   },
+        //   publishedAt: new Date(),
+        //   path: "/solutions/tari-comune-di-cagliari",
+        // },
```

Comment:
Did you left this comment on purpose?

Reply:
There was a bug when the stories' array had only one element in it, I commented these lines because of that, we can remove them or restore it, it makes no difference